pkg/epp/handlers/request.go

```diff
@@ -61,6 +61,35 @@ func (s *Server) HandleRequestBody(
 	loggerVerbose.Info("Model requested", "model", model)
 	modelName := model
 
+	// Resolve streaming options
+
+	streaming, ok := rb["stream"].(bool)
+	if !ok {
+		// streaming not set, no-op
+	} else {
+		reqCtx.Streaming = streaming
+	}
+
+	type Usage struct {
+		IncludeUsage string `json:"include_usage,omitempty"`
+	}
+	if streamOption, ok := rb["stream_options"]; ok {
```

Comment:
These logic should be under streaming= true

Reply:
Updated.